www/dohp02/SP2/js/app.js

```diff
@@ -0,0 +1,140 @@
+var cb = new Codebird;
+cb.setConsumerKey("7T3WDEIZI6AlJzuuPrufryPJk", "ChNjwMAe75zthLay6dwK5kpDqptuY6j3n9eEUF2WhHs9yy2qrX");
+cb.__call("oauth2_token", {}, function (reply, err) {
+    var bearer_token;
+    if (err) {
+        console.log("error response or timeout exceeded" + err.error);
+    }
+    if (reply) {
+        bearer_token = reply.access_token;
+    }
+});
+
+window.twttr = (function (d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0],
+        t = window.twttr || {};
+    if (d.getElementById(id)) return t;
+    js = d.createElement(s);
+    js.id = id;
+    js.src = "https://platform.twitter.com/widgets.js";
+    fjs.parentNode.insertBefore(js, fjs);
+
+    t._e = [];
+    t.ready = function (f) {
+        t._e.push(f);
+    };
+    return t;
+}(document, "script", "twitter-wjs"));
+
+function searchUsers() {
+    var params = {
+        screen_name: document.getElementById("search-input").value
+    };
+    document.getElementById("latest_container").innerHTML = "";
```

Comment:
opakovane vybirani elementu, casto dotcene elementy musis vybrat a ukladat do promennych a pak je v funkcich prepouzivat, take zkus pouzit na to jQuery misto nativnich getElementById

Reply:
Opraveno